src/util/std_expr.h

```diff
@@ -2315,7 +2315,7 @@ class extractbits_exprt:public exprt
     const exprt &_lower,
     const typet &_type):exprt(ID_extractbits, _type)
   {
-    copy_to_operands(_src, _lower, _upper);
+    copy_to_operands(_src, _upper, _lower);
   }
```

Comment:
A comment elsewhere says that this is the order that SMT-LIB uses. Otherwise I have no particular opinion about this...